examples/with-luis.ai/index.js

```diff
@@ -0,0 +1,27 @@
+const querystring = require('querystring');
+
+const axios = require('axios');
+
+const { LUIS_APP_KEY, LUIS_APP_ENDPOINT, LUIS_APP_ID } = process.env;
+
+module.exports = async function App(context) {
+  if (context.event.isText) {
+    const queryParams = {
+      verbose: true,
+      q: context.event.text,
+      'subscription-key': LUIS_APP_KEY,
+    };
+
+    const { data } = await axios.get(
+      `${LUIS_APP_ENDPOINT}luis/v2.0/apps/${LUIS_APP_ID}?${querystring.stringify(
+        queryParams
+      )}`
```

Comment:
https://github.com/axios/axios/blob/master/README.md#L88-L93
`{ params: queryParams }` is better in readability

Reply:
Thanks for kindly pointing this out. It is definitely way more readable.